### PR TITLE
chore: replace myFoo naming idiom with snake_case_

### PR DESCRIPTION
**Pull request by ckerr**
This comes out of the discussion at https://github.com/transmission/transmission/pull/1234#discussion_r427630467 to replace the `myFoo` member variable naming idiom with `foo_`.

Here's how it was done. I'm kind of happy about getting something this ugly to work, but I wouldn't say I'm proud of it... :smiling_imp: : 

```sh
$ grep --no-filename --only-matching --perl-regexp "my[A-Z][A-Za-z]+\w*" *\.h *\.cc | sort | uniq > oldvars
$ # process longer variable names first so that shorter names don't partially rename longer ones 
$ cat oldvars | awk '{ print length, $0 }' | sort --reverse --numeric-sort --stable | cut -d" " -f2- > oldvars
$ cat oldvars | sed --expression 's/\([A-Z]\)/_\L\1/g' --expression 's/$/_/' --expression 's/^my_//' > newvars
$ paste -d '/' oldvars newvars | sed --expression 's/^/"s\//' --expression 's/$/\/g"/' > expressions
$ # ... manually copyedit `expressions` to fix any edge cases ...
$ xargs -L1 -I '{}' perl -p -i -e '{}' *.cc *h < expressions
$ rm oldvars newvars expressions
```

### CODE REVIEW

File: qt/DetailsDialog.h [hunk 1]
```diff
@@ -89,20 +89,20 @@ private slots:
...
 
 private:
-    Session& mySession;
-    Prefs& myPrefs;
-    TorrentModel const& myModel;
+    Session& session_;
+    Prefs& prefs_;
+    TorrentModel const& model_;
 
     Ui::DetailsDialog ui;
```
Comment (mikedld) [inline qt/DetailsDialog.h:96]: Looks like you didn't catch the cases which didn't use `myXxx` scheme before.

File: qt/TorrentDelegateMin.cc [hunk 2]
```diff
@@ -49,8 +49,8 @@ namespace
 class ItemLayout
 {
 private:
-    QString myNameText;
-    QString myStatusText;
+    QString name_text_;
+    QString status_text_;
 
 public:
     QFont nameFont;
```
Comment (mikedld) [inline qt/TorrentDelegateMin.cc:56]: Do public fields get the same treatment or not? Maybe there shouldn't be any public fields?

File: qt/TrackerDelegate.cc [hunk 3]
```diff
@@ -29,13 +29,13 @@
 namespace
 {
 
-int const mySpacing = 6;
-QSize const myMargin(10, 10);
+int const spacing_ = 6;
+QSize const margin_(10, 10);
```
Comment (mikedld) [inline qt/TrackerDelegate.cc:33]: These aren't member variables.

File: qt/DetailsDialog.h (hunk 1 above)
Comment (ckerr) [reply qt/DetailsDialog.h]: tweaked in e5e8ffd2d91edb777af8564a885a4f2423c2043b

File: qt/TrackerDelegate.cc (hunk 3 above)
Comment (ckerr) [reply qt/TrackerDelegate.cc]: Fixed 8e466314f

File: qt/DetailsDialog.h (hunk 1 above)
Comment (mikedld) [reply qt/DetailsDialog.h]: That wasn't the only class with `ui` member. Didn't check if there're other (non-`ui`) cases out there.

File: qt/TorrentDelegateMin.cc (hunk 2 above)
Comment (ckerr) [reply qt/TorrentDelegateMin.cc:56]: I'd say yes, all fields should get the same treatment.

I wouldn't hold it as a _strict_ rule, but yeah as a guideline we should limit public / protected fields -- that's far out-of-scope for this PR though

**Comment by ckerr** [comment]
There's got to be a less manual / brittle way of doing this.

I'm going to close this PR and will do some research on tooling that might do this for us.

**Comment by xavery** [comment]
@ckerr If all else fails, this sounds like a job for a [custom clang-tidy check](https://clang.llvm.org/extra/clang-tidy/Contributing.html#writing-a-clang-tidy-check).

**Pull request closed without merging**